routers/user/profile.go

```diff
@@ -184,6 +184,7 @@ func Profile(ctx *context.Context) {
 				ctx.ServerError("GetRepositories", err)
 				return
 			}
+			models.RepositoryList(repos).LoadAttributes()
```

Comment:
@appleboy I think you should explain this line since maybe some think it's slower than before.